app.js

```diff
@@ -50,9 +52,6 @@ function initMap(){
 
     // New map
     var map = new google.maps.Map(document.getElementById('map'), options);
-
-
-
     // Loop through markers
     for(var i = 0;i < markers.length;i++){
```

Comment:
Similar comment as before, so in this case, would recommend using `let`

Reply:
Good catch, it's updated!